src/components/FloatingDialog.vue

```diff
@@ -6,9 +6,11 @@
       <el-header v-if="entry.mode==='normal'" style="text-align: left; font-size: 14px;padding:0" 
         height="40px" class="dialog-header">
         <DialogToolbarContent :dialogTitles="[indexTitle]"  @maximise="onMaximise" @minimise="onMinimise" 
-          @close="onClose"/>         
+          @close="onClose"/>
       </el-header>
+      
       <el-main class="dialog-main" :style="mainStyle">
+        <DatasetHeader v-if="entry.datasetTitle" class="dataset-header" :title="entry.datasetTitle" :url="entry.datasetUrl"></DatasetHeader>
```

Comment:
Have to take the height of the header into account. There is now overflow in the dalog. Create another div for the vuers with appropriate size. height: calc(100%-25px)?